app/controllers/oidc/pending_trusted_publishers_controller.rb

```diff
@@ -0,0 +1,65 @@
+class OIDC::PendingTrustedPublishersController < ApplicationController
+  include OIDC::Concerns::TrustedPublisherCreation
+
+  before_action :find_pending_trusted_publisher, only: %i[destroy]
+
+  def index
+    trusted_publishers = current_user
+      .oidc_pending_trusted_publishers.unexpired.includes(:trusted_publisher)
+      .order(:rubygem_name, :created_at).page(@page).strict_loading
+    render OIDC::PendingTrustedPublishers::IndexView.new(
+      trusted_publishers:
+    )
+  end
+
+  def new
+    pending_trusted_publisher = current_user.oidc_pending_trusted_publishers.new(trusted_publisher: OIDC::TrustedPublisher::GitHubAction.new)
+    render OIDC::PendingTrustedPublishers::NewView.new(
+      pending_trusted_publisher:
+    )
+  end
+
+  def create
+    trusted_publisher = current_user.oidc_pending_trusted_publishers.new(
+      create_params.merge(
+        expires_at: 7.days.from_now
```

Comment:
```suggestion
        expires_at: 12.hours.from_now
```

My thought here is that if you haven't been able to get it working in 12 hours, we want to close off access until you confirm (by creating a new pending publisher) that you are still trying to make it work.